src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped work!");
```

Comment:
The stop message should be consistent with the start message. Consider changing it to 'Excavator stopped working.' to match the informative style.